### DISCUSSION TITLE
Error saat Input Map di R Shiny

### DISCUSSION THREAD

**Discussion by defor27** [teamalgoritma/community#711] [Q&A] [answered]
[crime_shiny.zip](https://github.com/teamalgoritma/community/files/11936947/crime_shiny.zip)

Tidak dapat memasukkan map ke Dashboard R Shiny padahal ketika dicoba di rmd bisa tergenerate, Mohon bantuannya terima kasih

**Comment by jafarhf1** [comment, marked as answer]
Dear Bu @defor27,

Berdasarkan file yang diberikan, terdapat beberapa kode yang kurang tepat

1. Penggunaan fungsi `renderleaflet()` seharusnya dengan `renderLeaflet()` 
2. Penggunaan parameter label = labels pada fungsi addCircleMarkers() error karena tidak ada variabel labels tetapi yang ada adalah labels_map sehingga seharusnya adalah `addCircleMarkers(label = labels_map)`

Demikian yang dapat kami sampaikan, semoga membantu
